ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -0,0 +1,337 @@
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3client.integration;
+
+import com.google.common.collect.Lists;
+import com.spectralogic.ds3client.Ds3Client;
+import com.spectralogic.ds3client.commands.*;
+import com.spectralogic.ds3client.commands.spectrads3.*;
+import com.spectralogic.ds3client.helpers.Ds3ClientHelpers;
+import com.spectralogic.ds3client.helpers.options.WriteJobOptions;
+import com.spectralogic.ds3client.integration.test.helpers.TempStorageIds;
+import com.spectralogic.ds3client.integration.test.helpers.TempStorageUtil;
+import com.spectralogic.ds3client.models.*;
+import com.spectralogic.ds3client.models.bulk.Ds3Object;
+import com.spectralogic.ds3client.serializer.XmlProcessingException;
+import com.spectralogic.ds3client.utils.ResourceUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.*;
+import java.security.SignatureException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import static com.spectralogic.ds3client.helpers.Ds3ClientHelpers.*;
+import static com.spectralogic.ds3client.integration.Util.*;
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+import static org.junit.Assume.assumeThat;
+
+public class PutJobManagement_Test {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PutJobManagement_Test.class);
+
+    private static Ds3Client client;
+    private static final String TEST_ENV_NAME = "PutJobManagement_Test";
+    private static TempStorageIds envStorageIds;
+
+    @BeforeClass
+    public static void startup() throws IOException, SignatureException {
+        client = Util.fromEnv();
+        final UUID dataPolicyId = TempStorageUtil.setupDataPolicy(TEST_ENV_NAME, true, ChecksumType.Type.MD5, client);
+        envStorageIds = TempStorageUtil.setup(TEST_ENV_NAME, dataPolicyId, client);
+    }
+
+    @AfterClass
+    public static void teardown() throws IOException, SignatureException {
+        TempStorageUtil.teardown(TEST_ENV_NAME, envStorageIds, client);
+        client.close();
+    }
+
+    @Test
+    public void modifyJobPriority() throws IOException, SignatureException, XmlProcessingException {
+        final String bucketName = "test_modify_job_priority";
+        try {
+            client.putBucket(new PutBucketRequest(bucketName));
+
+            final List<Ds3Object> objects = new ArrayList<>();
+            final Ds3Object obj = new Ds3Object("test", 2);
```

Comment:
This can be shortened to `final List<Ds3Object> objects = Lists.newArrayList(new Ds3Object("test", 2));`